src/mono/mono/metadata/icall-decl.h

```diff
@@ -304,4 +304,8 @@ ICALL_EXPORT gint32   ves_icall_System_Threading_LowLevelLifoSemaphore_TimedWait
 ICALL_EXPORT void     ves_icall_System_Threading_LowLevelLifoSemaphore_ReleaseInternal (gpointer sem_ptr, gint32 count);
 #endif
 
+#if defined(ENABLE_NETCORE)
```

Comment:
Shouldn't this be behind `defined(TARGET_AMD64)` as well?

Reply:
Yes.